modules/ROOT/pages/security/single-sign-on.adoc

```diff
@@ -25,20 +35,35 @@ It *does not* give access to edit the project settings, for example to edit the
 Log-in methods are different for each SSO level.
 Administrators can configure a combination of one or more of the log-in methods.
 
-*Organization-level supports:*
+*Supported log-in methods at the organization-level:*
 
 * Email/password
 * Okta
 * Microsoft Entra ID
 * Google SSO (not Google Workspace SSO)
 
-*Project-level supports:*
+*Supported log-in methods at the project-level:*
 
 * User/password
 * Okta
 * Microsoft Entra ID
 
-At the project-level admins cannot disable user/password, but at the organization-level admins can disable email/password and Google SSO as long as there is at least one other custom SSO provider configured.
+At the project-level, admins cannot disable user/password. 
+However, at the organization-level, admins can disable email/password and Google SSO as long as at least one other custom SSO provider is configured.
+
+== Pilot testing
+
+*Organization-level testing*
+
+Keep the Email/password log-in method enabled, so that if SSO fails, you can still access the Aura console and adjust the configuration.
+
+Login to the Aura console with SSO.
+
+*Project level-testing*
+
+Keep the User/password login enabled, so that if SSO fails, you can still access the Aura console and adjust the configuration.
+
+Create a new instance in the project with SSO and login.
```

Comment:
This seems out of place. Maybe stick it in as an admonition in the section on Create a new SSO config?